corehq/apps/app_manager/models.py

```diff
@@ -4572,6 +4646,14 @@ def get_hierarchy(case_type):
 
         return meta
 
+    def get_subcase_types(self, case_type):
+        """
+        Return the subcase types defined across an app for the given case type
+        """
+        return {t for m in self.get_modules()
+                if m.case_type == case_type
+                for t in m.get_subcase_types()}
```

Comment:
This won't necessarily work correctly since Advanced forms can have different case types to the module.